presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -1066,7 +1067,12 @@ public SystemSessionProperties(
                         QUERY_OPTIMIZATION_WITH_MATERIALIZED_VIEW_ENABLED,
                         "Enable query optimization with materialized view",
                         featuresConfig.isQueryOptimizationWithMaterializedViewEnabled(),
-                        true));
+                        true),
+                booleanProperty(
+                        AGG_IF_TO_FILTER_REWRITE_ENABLED,
+                        "Enable rewriting AGG(IF()) to AGG() FILTER",
```

Comment:
Enable rewriting the if expression inside an aggregation function to a filter clause outside aggregation.